ENIGMAsystem/SHELL/Universal_System/Extensions/DataStructures/data_structures.cpp

```diff
@@ -50,10 +50,11 @@ template<typename t> bool tequal(t v1, t v2) { return v1 == v2; }
 template<> bool tequal(float v1, float v2)   { return fequal(v1, v2); }
 template<> bool tequal(double v1, double v2) { return fequal(v1, v2); }
 
-namespace enigma {
-  static inline int random_integer(int x) {
-    return int(enigma_user::random(x));
-  }
+template<class RandomIt>
+void mt_random_shuffle(RandomIt first, RandomIt last) {
```

Comment:
I see no need for this to be a template and is overall bad change because it generates a new seed every time you call it

Reply:
i see no need for you to be a contributor to this project and is overall a bad change because you scared away every contributor we've ever had other than me and Robert. :P